x-pack/solutions/security/plugins/security_solution/public/cases/components/case_events/search_events.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EcsFlat } from '@elastic/ecs';
+import type { DataView } from '@kbn/data-views-plugin/common';
+import { AbortError } from '@kbn/kibana-utils-plugin/common';
+import type { SortColumnTable } from '@kbn/securitysolution-data-table';
+import type { EcsSecurityExtension } from '@kbn/securitysolution-ecs';
+import type { TimelineItem } from '@kbn/timelines-plugin/common';
+import { lastValueFrom } from 'rxjs';
+import { KibanaServices } from '../../../common/lib/kibana';
+
+export const searchEvents = async (
+  signal: AbortSignal | undefined,
+  dataView: DataView | undefined,
```

Comment:
dataView is typed as `DataView` in `use_get_event`, undefined is not needed here